### PR TITLE
Observation Noise

### PR DESCRIPTION
**Pull request by adam2392**
Added observation noise for the Projections class.

Marmaduke, I switched some implementations from our original convo. Let me know if they are incorrect.

Note:
- I switched to the projections class because it did not make sense how to add noise to the Monitors class, if the monitors for the source signals would already have noise. 
- some issues I am not sure of on regarding the dt, shape, nsig terms in setting the observation noise, and configuration.

### CODE REVIEW

**Comment by maedoc** [comment]
hi @adam2392 is there a related notebook or demo for this? if not, consider adding one to the [docs demo folder](https://github.com/the-virtual-brain/tvb-documentation/tree/master/demos), and link to it here.

**Comment by adam2392** [comment]
Created a pull request within the tvb-documentation. The corresponding doc should be at this directory:

https://github.com/the-virtual-brain/tvb-documentation/tree/master/demos

Let me know if there is anything else needed here.

File: tvb/simulator/monitors.py [hunk 1]
```diff
@@ -514,6 +521,21 @@ def sample(self, step, state):
...
+            '''
+            1. Should dt be the same as in the source level?
+            2. how should shape be set here for colored noise?
+            '''
+            if self.obsnoise is not None:
+                # configure the noise level
+                if self.obsnoise.ntau > 0.0:
+                    self.obsnoise.configure_coloured(self.dt, sample.shape)
+                else:
+                    self.obsnoise.configure_white(self.dt)
```
Comment (maedoc) [inline tvb/simulator/monitors.py]: configuration should be done in the monitor's configure step if possible.  

> Should dt be the same as in the source level?

yes

> how should shape be set here for colored noise?

looks correct to me

**Comment by maedoc** [comment]
@adam2392 if you need to pull this contrib also please close the PR

**Comment by adam2392** [comment]
I don't need to close this I believe. This was just an additional few lines of code to allow user to add noise generated post-simulation.

**Comment by maedoc** [comment]
I think this is still a good addition, so I'm going to merge this.  Thanks @adam2392

**Pull request merged**